k8s/templates/business-edit-ui.dc-srvc-rt-cmap-kcmap.json

```diff
@@ -1,451 +0,0 @@
-{
```

Comment:
Also, in a future tech review, can you please show us where config data is kept? Config map? One Password? Other?

Reply:
Yes, it will all in 1password. Next CD flow release will include the process to generate the configmaps in each deployment.

Reply:
@severinbeauvais those are different things. 
Source of truth is still 1Password, that hasn't changed.
Pods will get mapped to that via configmaps/secrets.